nemo/collections/asr/parts/preprocessing/segment.py

```diff
@@ -181,6 +185,8 @@ def from_file(
         trim_hop_length=512,
         orig_sr=None,
         channel_selector=None,
+        normalize_db=False,
+        normalize_db_target=-20.0,
     ):
```

Comment:
The normalization feature must have thorough unit tests since `segment.py` is used by all ASR related NeMo modules. 

Be sure to add normalization tests and also, it needs to handle multi-channel capability to normalize multiple files together.